packages/xstate-lit/src/useActorRef.ts

```diff
@@ -0,0 +1,9 @@
+import { Actor, ActorOptions, AnyActorLogic, createActor } from 'xstate';
+
+export const useActorRef = <TMachine extends AnyActorLogic>(
```

Comment:
this one should be exported from the index and it should also accept `TLogic` (not just `TMachine`)